benchmarks/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(gups)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(launch_latency)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(stream)
+KOKKOS_ADD_BENCHMARK_DIRECTORIES(copy_view_performance)
```

Comment:
This is not sufficient. You need to add CMakeLists.txt in the new repo you added.
Isn't CMake warning about this?

Reply:
No it actually created the executable.

Reply:
It did not work in any of our CI builds which enable benchmarks. Note you likely had an old executable build with your makefile?

Reply:
Sorry about this. I actually forgot to git add `CMakeLists.txt` to the repo.